examples/events/README.md

```diff
@@ -292,4 +111,129 @@ function computeDropzoneProps(props, node) {
 <Parent />
 ```
 
-Note that the sort of behavior illustrated above can lead to some confusion. For example, the `onDragLeave` callback of the parent will not be called if the callback for the same event is invoked on the child. This happens because we invoked `stopPropagation()` on the event from the child.
+Note that internally we use `event.stopPropagation()` to achieve the behavior illustrated above, but this comes with its own [drawbacks](https://javascript.info/bubbling-and-capturing#stopping-bubbling).
+
+If you'd like to selectively turn off the default dropzone behavior for `onClick`, `onKeyDown` (both open the file dialog), `onFocus` and `onBlur` (sets the `isFocused` state) and drag events; use the `{noClick}`, `{noKeyboard}` and `{noDrag}` properties:
+```jsx harmony
+import React, {useCallback, useReducer} from 'react';
+import {useDropzone} from 'react-dropzone';
+
+const initialEvtsState = {
+  noClick: true,
+  noKeyboard: true,
+  noDrag: true,
+  files: []
+};
+
+function Events(props) {
+  const [state, dispatch] = useReducer(reducer, initialEvtsState);
+  const createToggleHandler = type => () => dispatch({type});
+
+  const onDrop = useCallback(files => dispatch({
+    type: 'setFiles',
+    payload: files
+  }), []);
+
+  const {getRootProps, getInputProps, isFocused} = useDropzone({...state, onDrop});
+  const files = state.files.map(file => <li key={file.path}>{file.path}</li>);
+
+  const options = ['noClick', 'noKeyboard', 'noDrag'].map(key => (
```

Comment:
While I appreciate the sophisticated use of react hooks like `useCallback` and the meta programming on the options, I think the goal of a README.md is readability. Could we emphasize a simple use case and allow readers to extrapolate from that one scenario to all other options? I believe you touched on this with a `noDragEventsBubbling` option.

Reply:
I can definitely simplify some of the examples and perhaps create some codesandbox examples and just link to them instead.